src/pfs/pfs.proto

```diff
@@ -131,36 +131,25 @@ message CommitInfo {
   google.protobuf.Timestamp started = 6;
   google.protobuf.Timestamp finished = 7;
   uint64 size_bytes = 8;
-
-  // the commits and their original branches on which this commit is provenant
-  repeated CommitProvenance provenance = 9;
-
-  // ReadyProvenance is the number of provenant commits which have been
-  // finished, if ReadyProvenance == len(Provenance) then the commit is ready
-  // to be processed by pps.
-  int64 ready_provenance = 10;
-
-  repeated CommitRange subvenance = 11;
-
-  int64 subvenant_commits_success = 12;
-  int64 subvenant_commits_failure = 13;
-  int64 subvenant_commits_total = 14;
 }
 
 message Job {
   string id = 1 [(gogoproto.customname) = "ID"];
 }
 
+message JobCommitInfo {
+  Branch branch = 1;
+  repeated Branch provenance = 2;
+}
+
 message StoredJobInfo {
   Job job = 1;
-  CommitOrigin origin = 2;
-  repeated Commit commits = 3;
+  repeated JobCommitInfo job_commits = 3;
 }
 
 message JobInfo {
   Job job = 1;
-  CommitOrigin origin = 2;
-  repeated CommitInfo commits = 3;
+  repeated JobCommitInfo job_commits = 3;
 }
```

Comment:
Do we need these as separate messages now? It looks like they have identical definitions.

Reply:
Like we discussed, the hydrated one will be filled out in `InspectJob` so its fields will be expanded to account for that.  I don't think there is much of an internal use-case for an un-hydrated `JobInfo`.